doc/Writerside/topics/starter-topic.md

```
@@ -0,0 +1,12 @@
+# Подсистема интеграции.
+
+Данная подсистема интеграции позволяет быстро и просто организовать событийную интеграцию между 1с и другими информационными системами.
+
+## Основные преимущества.
```

Markdownlint

Punctuation: '.'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)